src/NzbDrone.Core/Indexers/TorrentRssIndexer/TorrentRssIndexerSettings.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Indexers.TorrentRssIndexer
+{
+    public class TorrentRssIndexerSettingsValidator : AbstractValidator<TorrentRssIndexerSettings>
+    {
+        public TorrentRssIndexerSettingsValidator()
+        {
+            RuleFor(c => c.BaseUrl).ValidRootUrl();
+        }
+    }
+
+    public class TorrentRssIndexerSettings : IProviderConfig
+    {
+        private static readonly TorrentRssIndexerSettingsValidator validator = new TorrentRssIndexerSettingsValidator();
+
+        public TorrentRssIndexerSettings()
+        {
+            BaseUrl = string.Empty;
+            ValidEntryPercentage = 75;
+            ValidSizeThresholdMegabytes = 12;
+        }
+
+        [FieldDefinition(0, Label = "Full RSS Feed URL")]
+        public String BaseUrl { get; set; }
+
+        [FieldDefinition(1, Advanced = true, Label = "Valid Entry Percentage", HelpText = "Percentage of entries in the RSS feed which need to be parseable")]
+        public int ValidEntryPercentage { get; set; }
+
+        [FieldDefinition(2, Advanced = true, Label = "Valid Size Threshold (MB)", HelpText = "Threshold of Parsed Torrent Size to consider it valid")]
+        public int ValidSizeThresholdMegabytes { get; set; }
```

Comment:
Hardcode the limit at 2MB. Torrent files normally will never reach that. Only very badly created .torrent files (too small piece size) would even come close.